ui/src/components/admin/forms/AdminFormDataViewer.js

```diff
@@ -0,0 +1,252 @@
+import React, { useEffect, useState } from 'react';
+import {} from '@chakra-ui/react';
+import { customAxios as axios } from '../../helpers/customAxios';
+import { AgGridReact } from 'ag-grid-react';
+import 'ag-grid-community/dist/styles/ag-grid.css';
+import 'ag-grid-community/dist/styles/ag-theme-alpine.css';
+import { Button, Heading, Text, Input, HStack, Box } from '@chakra-ui/react';
+import { DateTime } from 'luxon';
+
+export default function AdminFormDataViewer(props) {
+  const {
+    location: { state },
+  } = props;
+  const formName = state.name;
+  const formId = state.id;
+
+  const [formData, setFormData] = useState([]);
+  const [api, setApi] = useState();
+  const [startDate, setStartDate] = useState('');
+  const [endDate, setEndDate] = useState('');
+
+  const getFilterType = () => {
+    if (startDate !== '' && endDate !== '') return 'inRange';
+    else if (startDate !== '') return 'greaterThan';
+    else if (endDate !== '') return 'lessThan';
+  };
+
+  const exportHandler = () => {
+    if (api) {
+      api.exportDataAsCsv(exportParams());
+    }
+  };
+  const exportParams = () => {
+    let now = DateTime.now();
+    let fileName = `${now.toFormat('yyyyMMdd_HHmmss')}_${formName}.csv`;
+
+    return {
+      skipColumnGroupHeaders: true,
+      allColumns: true,
+      fileName: fileName,
+    };
+  };
+
+  const dateFilterParams = {
+    comparator: function (filterLocalDateAtMidnight, cellValue) {
+      let dateAsString = cellValue;
+      if (dateAsString == null) return -1;
+
+      let dateAtMidnight = DateTime.fromJSDate(filterLocalDateAtMidnight);
+      let cellDate = DateTime.fromFormat(dateAsString, 'yyyy-MM-dd HH:mm:ss');
+      if (cellDate === dateAtMidnight) {
+        return 1;
+      }
+      if (cellDate < dateAtMidnight) {
+        return -1;
+      }
+      if (cellDate >= dateAtMidnight) {
+        return 1;
+      }
+    },
+    browserDatePicker: true,
+  };
+
+  useEffect(() => {
+    const getFormData = async () => {
+      try {
+        const { data } = await axios.get('/api/forms/get-submission', {
+          params: {
+            formId: formId,
+          },
+        });
+        let formDataTemp = [];
+        data.forEach(function (item) {
+          let temp = {};
+          temp = item.submissionData;
+
+          //format date to: yyy-mm-dd hh:mm:ss
+          let dateTimeRaw = DateTime.fromISO(item.updatedAt);
+          temp.updatedAt = dateTimeRaw
+            .toISO()
```

Comment:
You can do `dateTimeRaw.toFormat('yyyy-MM-dd HHmmss');`


Or wtv format u need instead of replace